test/integration/tasks.test.js

```diff
@@ -1460,87 +1470,4 @@ describe("Tasks", function () {
       expect(tasksLogs.body.error).to.be.equal("Error: Error occurred");
     });
   });
-
-  describe("PATCH /tasks/:id should update the tasks by SuperUser", function () {
-    beforeEach(async function () {
-      const superUserId = await addUser(superUser);
-      superUserJwt = authService.generateAuthToken({ userId: superUserId });
-
-      await firestore.collection("tasks").doc("4kAkRv9TBlOfR6WEUhoQ").set({
-        assignee: "SooJK37gzjIZfFNH0tlL",
-        status: "IN_PROGRESS",
-        percentCompleted: 80,
-        startedOn: 1701388800000,
-        endsOn: 1701561600000,
-      });
-    });
-
-    afterEach(async function () {
-      await firestore.collection("tasks").doc("4kAkRv9TBlOfR6WEUhoQ").delete();
-    });
-
-    it("Should unassign the task with other fields reset exclusively", async function () {
-      const res = await chai
-        .request(app)
-        .patch(`/tasks/4kAkRv9TBlOfR6WEUhoQ`)
-        .set("cookie", `${cookieName}=${superUserJwt}`)
-        .send({
-          assignee: null,
-          status: TASK_STATUS.AVAILABLE,
-          percentCompleted: 0,
-          startedOn: null,
-          endsOn: null,
-        });
-
-      expect(res).to.have.status(204);
-
-      const docSnapshot = await firestore.collection("tasks").doc("4kAkRv9TBlOfR6WEUhoQ").get();
-
-      const updatedData = docSnapshot.data();
-      expect(updatedData.assignee).to.equal(null);
-      expect(updatedData.status).to.equal(TASK_STATUS.AVAILABLE);
-      expect(updatedData.percentCompleted).to.equal(0);
-      expect(updatedData.startedOn).to.equal(null);
-      expect(updatedData.endsOn).to.equal(null);
-    });
-
-    it("Should unassign the task with other fields reset internally", async function () {
-      const res = await chai
-        .request(app)
-        .patch(`/tasks/4kAkRv9TBlOfR6WEUhoQ`)
-        .set("cookie", `${cookieName}=${superUserJwt}`)
-        .send({
-          status: TASK_STATUS.AVAILABLE,
-        });
-
-      expect(res).to.have.status(204);
-
-      const docSnapshot = await firestore.collection("tasks").doc("4kAkRv9TBlOfR6WEUhoQ").get();
-
-      const updatedData = docSnapshot.data();
-      expect(updatedData.assignee).to.equal(null);
-      expect(updatedData.status).to.equal(TASK_STATUS.AVAILABLE);
-      expect(updatedData.percentCompleted).to.equal(0);
-      expect(updatedData.startedOn).to.equal(null);
-      expect(updatedData.endsOn).to.equal(null);
-    });
```

Comment:
why did you remove existing test cases from the codebase? It is intended?

Reply:
yes this is intended . 
This is the piece of code that was removed https://github.com/Real-Dev-Squad/website-backend/pull/1875/files#r1478565929